class.jetpack.php

```diff
@@ -3215,26 +3227,330 @@ function admin_connect_notice() {
 		if ( ! current_user_can( 'jetpack_connect' ) )
 			return;
 
-		$dismiss_and_deactivate_url = wp_nonce_url( Jetpack::admin_url( '?page=jetpack&jetpack-notice=dismiss' ), 'jetpack-deactivate' );
+		$dismiss_and_deactivate_url = wp_nonce_url(
+			Jetpack::admin_url( '?page=jetpack&jetpack-notice=dismiss' ),
+			'jetpack-deactivate'
+		);
 		?>
+
+		<div id="message" class="updated jp-wpcom-connect__container">
```

Comment:
Instead of adding 200+ LOC in an already huge file, why don't we abstract this to it's own view file and include it here?